_po/ja/news/index.po

```diff
@@ -21,38 +21,126 @@ msgstr ""
+"h as PostgreSQL Licens, BSD License, and MIT License."
+msgstr ""
+"Supabaseさんは、PGroongaがPostgreSQL Licenseで提供されているソフトウェアであるという点に魅力を感じてくれました。Supab"
+"aseさんも、PostgreSQL LicenseやBSD License、MIT Licenseのソフトウェアを採用してサービスを提供しオープンソースソフ"
+"トウェアへ貢献しています。"
+
+msgid ""
+"Also Supabase give financial contributions to various OSS including PGroonga v"
+"ia [Open Collective](https://opencollective.com/), a fundraising + legal statu"
+"s + money management platform."
+msgstr ""
+"また、オープンソースソフトウェアのエコシステムの概念を実践して、PGroongaを含めた様々なオープンソースソフトウェアに対しての[Open Collect"
+"ive](https://opencollective.com/)というサービスをつかった資金サポートもしています。"
+
+msgid ""
+"* [Supabase Open Collective Page](https://opencollective.com/supabase)\n"
+"* [PGroonga Open Collective Page](https://opencollective.com/pgroonga)"
+msgstr ""
+
+msgid ""
+"It is a great thing for our community that OSS contribution leads this coopera"
+"tion."
+msgstr "このように、オープンソースへの貢献を通じてコラボレーションできるのは素晴らしいことですね！"
+
+msgid "## 2.4.3: 2022-12-21 {#version-2-4-3}"
 msgstr ""
 
 msgid "### Improvements"
 msgstr "### 改良"
 
 msgid ""
-"  * Added support for PostgreSQL 15 on Windows.[GitHub#256][Reported by Raif A"
-"tef]"
+"  * [[`pgroonga_highlight_html` function][highlight-html]] Added support for t"
+"he `text[]` type for `target`."
 msgstr ""
 
-msgid "  * Added support for PostgreSQL 15 on Ubuntu 22.04."
+msgid "    `pgroonga_highlight_html` only supported the `text` type for `target`."
+msgstr ""
+
+msgid "    We can now specify the `text[]` type for `target`."
 msgstr ""
 
 msgid ""
-"  * Added a new module [`pgroonga_standby_maintainer` module][pgroonga-standby"
-"-maintainer] executing ``pgroonga_wal_apply()`` and ``pgroonga_vacuum()`` auto"
-"matically on a standby database."
+"    A type of a returned value is also `text[]` when we specify the `text[]` t"
+"ype for `target`."
 msgstr ""
 
 msgid ""
-"  * [`pgroonga_snippet_html` function][snippet-html] Added a new argument, ``p"
-"groonga_snippet_html``, specifing snippet length dynamically.[GitHub#253][Repo"
-"rted by askdkc][GitHub#255][Patched by askdkc]"
+"    ```sql\n"
+"    SELECT pgroonga_highlight_html(\n"
+"      ARRAY['one two three', NULL, 'five', 'six three'],\n"
+"      ARRAY['two', 'six']);\n"
+"    --                                     pgroonga_highlight_html            "
+"                            \n"
+"    -- -----------------------------------------------------------------------"
+"--------------------------------\n"
+"    -- {\"one <span class=\\\"keyword\\\">two</span> three\",NULL,five,\"<span class="
+"\\\"keyword\\\">six</span> three\"}\n"
+"    -- (1 row)\n"
+"    ```"
 msgstr ""
 
 msgid "### Fixes"
 msgstr "### 修正"
 
 msgid ""
-"  * Fixed a bug that ``VACUUM`` fails while we execute 2 phase commit. [GitHub"
-"#252][Reported by Raif Atef]"
+"  * Published the latest [Docker image](https://hub.docker.com/r/groonga/pgroo"
+"nga)"
+msgstr ""
+
+msgid "    We had not published Docker images since 2.3.9."
```

Comment:
もし、やめた　という事であれば　stopped publishが自然です。

これまで一度も出したことがないのならhad not でいいですがこのsince とDocker imagesのバージョン情報などがないので、2.3.9から一切のDocker Imageをpublishしていないという意味になりそうですが大丈夫でしょうか。

Reply:
前提として、バージョンアップをする度にそのバージョンのDocker imageを公開することになっています。
なのですが、2.3.9をリリースしたときから現在まで（2.4.3）公開していませんでした。
公開したつもりになっていたのですが、公開できていなかったというミスです。

というわけでして、2.3.9から公開すべきDocker Imageを公開し忘れていた、というのがここで伝えたい意図です。

Reply:
おっしゃる意図であれば、下記がいいかと思います。

```
The matched Docker images was not published between 2.3.9.  and current version.

```

PGroongaとDocker Imageがそのバージョンごとにマッチしているものがあるはずというのは前提だと思うので、
対応するDocker imageが2.3.9から現在のバージョンでは公開されていません。　という形です。

Reply:
Thank you. I have fixed.